CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   ([#476](https://github.com/jjriv/emogrifier/pull/476))
 
 ### Changed
+- Add type hint checking to the code sniffs
+  ([#563](https://github.com/MyIntervals/emogrifier/issues/563),
+  [#566](https://github.com/MyIntervals/emogrifier/pull/566))
```

Comment:
The link to the PR should be enough in the changelog. You can add "FIxes #xy" as a single line in the commit message to relate to the ticket (and to autoclose it on  merge).